ai_edge_torch/generative/examples/stable_diffusion/diffusion.py

```diff
@@ -250,6 +250,7 @@ def __init__(self, config: unet_cfg.DiffusionModelConfig):
 
     attention_config = layers_cfg.AttentionConfig(
         num_heads=config.transformer_num_attention_heads,
+        head_dim=block_out_channels[0] // config.transformer_num_attention_heads,
```

Comment:
Please refer to https://github.com/google-ai-edge/ai-edge-torch/pull/87/files#r1678556679, if #87 is not going to be submitted.
Same for the configs in stable_diffusion_loader.

Reply:
#87 is now closed in favor of #120 which this PR is based on.